index.html

```diff
@@ -0,0 +1,720 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/793d8924a3.js" crossorigin="anonymous"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap">
+    <title>TNW</title>
+</head>
+
+<body>
+    <nav class="flex">
+        <ul class="flex">
+            <li class="color-y">News</li>
+            <li class="color-y">Events </li>
+            <li class="color-y">Business </li>
+            <li class="color-y">AMAs</li>
+            <li class="color-y">Spaces</li>
+            <li class="color-y">Terms & Conditions</li>
+        </ul>
+        <ul class="flex top-right">
+            <li class="color-y">About</li>
+            <li class="color-y">Advertise</li>
+            <li class="color-y">Jobs</li>
+            <li class="color-y">Contact</li>
+        </ul>
+    </nav>
```

Comment:
- [ ] Kindly recheck this. You have bugs in many screen sizes. I think the problem lies in the containers, and many of the content seems to be overflowing and not behaving the way they should.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/89556710-2d436f00-d7e8-11ea-9f94-cee779e34833.png)|![image](https://user-images.githubusercontent.com/61479648/89556774-43512f80-d7e8-11ea-95e0-49f4ebff3b9a.png)  |

- [ ] the black border should have full width

![image](https://user-images.githubusercontent.com/61479648/89556897-6976cf80-d7e8-11ea-9226-2d8f12d56e3d.png)